### PR TITLE
LINTKA-381: Style all components and pages to prepare for deployment

### PR DESCRIPTION
**Pull request by ChTiSh**
Style all components preparing for deployment
Configured 3 screen sizes

image
image
image

## Description

Fixes # (issue) LINKTA-381

## Type of change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 Documentation update
- [ ] Other 🗒️ **\*\***\_\_**\*\***

## How Has This Been Tested?

To Test all stylings:
1. Go to localhost:3000 - Home Page (3 screen sizes)
2. Fill out the form - Form States/Error
3. Click Join Waiting list - Check EmailVerificationPrompt
4. Go to Email Click Sign in - Check Thank you Page on localhost:3000/thank-you
5. Go to localhost:3000/not-found - not found page
6. Throw an error on /legal/page.tsx by ```throw new Error ('testing') to check error.tsx

## Checklist:

- [X] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [ ] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [ ] I have ensured that my pull request title is descriptive.

### CODE REVIEW

File: app/components/main-content/PrelaunchSignUpForm.tsx
Comment (ejwoods) [inline app/components/main-content/PrelaunchSignUpForm.tsx:140]: The use of a `<section>` element here to style the button and footer implies that it's a standalone section of content. I would use a simple `<div>` to implement this styling instead.

File: app/components/main-content/PrelaunchSignUpForm.tsx
Comment (ejwoods) [inline app/components/main-content/PrelaunchSignUpForm.tsx:141]: Similarly, this `<section>` element is a remnant from when there was an OAuth sign-in section. I would use a `<div>` here as well for styling if needed, but I'm not sure the styles are doing anything here - I didn't notice a change when removing them. I'd also argue the `aria-label` isn't needed anymore.

File: app/components/common/UniversalButton.tsx
Comment (kinskiwu) [inline app/components/common/UniversalButton.tsx:1]: Looks like this component overwrote the changes on dev.

File: app/components/main-content/KeyValueProposition.tsx
Comment (kinskiwu) [inline app/components/main-content/KeyValueProposition.tsx]: `help` -> `Help`

File: app/not-found.tsx
Comment (kinskiwu) [inline app/not-found.tsx:21]: Looks like this component also overwrote the changes on dev.

File: app/page.tsx
Comment (kinskiwu) [inline app/page.tsx:25]: Would be better for readibility if we edit the jsdoc to explain the setTimeout logic change

File: app/components/main-content/ThankYou.tsx
Comment (kinskiwu) [inline app/components/main-content/ThankYou.tsx]: Looks like this component also overwrote the changes on dev.

File: app/error.tsx
Comment (kinskiwu) [inline app/error.tsx]: `Page` -> `page`

File: app/error.tsx
Comment (kinskiwu) [inline app/error.tsx]: `Contact` -> `contact`

File: public/lottiefiles/loading-linkta.json
Comment (kinskiwu) [inline public/lottiefiles/loading-linkta.json]: This looks good. We can create a backlog ticket to add a loader for the home page with this lottie.

File: app/components/main-content/ThankYou.tsx
Comment (kinskiwu) [reply app/components/main-content/ThankYou.tsx]: It has quite some styling involved so I'll leave the update to you.

**Review by kinskiwu — APPROVED** [review]
Thanks for the hard work! Please resolve the conversation before merging.

File: public/lottiefiles/loading-linkta.json
Comment (ChTiSh) [reply public/lottiefiles/loading-linkta.json]: Thank you Kinski, your loader actually looks really good as well

File: app/components/main-content/ThankYou.tsx
Comment (ChTiSh) [reply app/components/main-content/ThankYou.tsx]: Just look through the file, the changes were intentional as thank you page was built based off an older version of the submission status page. Since this PR was intended to merge last, we decided to overwrite this page with the last PR.

File: app/page.tsx
Comment (ChTiSh) [reply app/page.tsx:25]: Changed it, thank you Kinski for pointing that out

File: app/components/main-content/ThankYou.tsx
Comment (kinskiwu) [reply app/components/main-content/ThankYou.tsx:32]: Please double check the content. This current version contains typos which was corrected in the content being overwritten.

Comment (kinskiwu) [reply app/components/main-content/ThankYou.tsx]: Email -> email

Comment (ChTiSh) [reply app/components/main-content/ThankYou.tsx]: Oh my eyes... I kept checking the email address and couldn't find the typo. Thank you @kinskiwu

**Review by ejwoods — APPROVED** [review]
Looks good - thank you for all your work on this! Approving so we can make new tickets to address further issues rather than address them here 👍 

Just gonna leave a couple comments I wrote when I started a review last week so they can be referenced later.

**Comment by ChTiSh** [comment]
Thank you @ejwoods

**Pull request merged**
